tests/visualization_tests/matplotlib_tests/test_slice.py

```diff
@@ -81,19 +98,19 @@ def test_plot_slice_log_scale() -> None:
 
     # Plot a parameter.
     figure = plot_slice(study, params=["y_log"])
-    assert len(figure.get_lines()) == 0
+    assert len(figure.findobj(PathCollection)) == 1
```

Comment:
In addition to the number of objects, we may want to assert the values using `allclose`, both for test accuracy and consistency with `test_slice_plot`.

Reply:
The plotly `test_slice.py` does not have accuracy tests as well, so do you think we should modify both of plotly/matplotlib in this part?

Reply:
Ah, sorry; just applying my comment causes inconsistency between the matplotlib and plotly tests.

As you mention, adding the same tests to plotly one should be the best way.